src/w3d/renderer/render2dsentence.cpp

```diff
@@ -274,15 +508,23 @@ void FontCharsClass::Create_GDI_Font(const char *font_name)
     m_ascent = text_metric.tmAscent;
     m_charHeight = text_metric.tmHeight;
     m_overhang = text_metric.tmOverhang;
-
+#endif
+#endif
     if (is_generals) {
         m_overhang = 0;
     }
-#endif
 }
 
 void FontCharsClass::Free_GDI_Font()
 {
+#ifdef BUILD_WITH_FREETYPE
```

Comment:
I suggest to make this function call 2 sub functions

Free_Windows_Font
Free_Freetype_Font

This way it becomes slimmer.

```c++
void FontCharsClass::Free_GDI_Font()
{
#if defined(BUILD_WITH_FREETYPE)
  Free_Freetype_Font();
#else defined(PLATFORM_WINDOWS)
  Free_Windows_Font()
#endif
}
```